client/src/components/BannerImageCard/BannerImageCard.css

```diff
@@ -0,0 +1,46 @@
+.BannerImageCard {
+    padding-top: 5rem;
+    margin: 0 auto;
+    width: 100%;
+    width: 100vw;
+    min-height: 230px;
+    background-position: center;
+    background-size: cover;
+    background-repeat: no-repeat;
+    text-align: right;
+    font-family: var(--default-font);
+    font-size: 48px;
+    color: #fafafa;
+    line-height: 72px;
+}
+
+.BannerImageCard__text{
+    margin: 0; 
+    margin-top: 8rem;
+    margin-right: 2rem;
+}
+
+@media only screen and (max-width: 576px) {
+    .BannerImageCard__text {
+        margin: 0 auto;
+        margin-top: 3rem;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-content: center;
+        text-align: center;
+    }
+}
+
+@media only screen and (max-width: 320px) {
+    .BannerImageCard__text {
+        margin: 0 auto;
+        margin-top: 2rem;
+        display: flex;
+        flex-direction: column;
+        justify-content: center;
+        align-content: center;
+        text-align: center;
+        font-size: 3rem;
+    }
+}
```

Comment:
Remove the styles here that are repeated from the other media query. They are only needed in one place - when you use max-width, the style apply to the specified screen size and smaller.